var/spack/repos/builtin/packages/tppred/package.py

```diff
@@ -0,0 +1,54 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class Tppred(Package):
+    """TPPRED is a software package for the prediction of mitochondrial
+       targeting peptides from protein primary sequence."""
+
+    homepage = "https://tppred2.biocomp.unibo.it/tppred2/default/software"
+    url      = "http://biocomp.unibo.it/savojard/tppred2.tar.gz"
+
+    version('2.0', 'cd848569f6a8aa51d18fbe55fe45d624')
+
+    depends_on('python@2.7:2.999', type='run')
+    depends_on('py-scikit-learn@0.13.1', type='run')
+    depends_on('emboss')
+
+    def url_for_version(self, version):
+        url = 'http://biocomp.unibo.it/savojard/tppred{}.tar.gz'
```

Comment:
Replace `{}` with `{0}`. The former only works in Python 2.7+, the latter works in Python 2.6+.